packages/experiments/src/components/signals/Signals.scss

```diff
@@ -171,9 +171,9 @@ i.emailed {
 }
 
 .bookmarkOutline {
-  color: $ms-color-black;
+  color: $ms-color-neutralSecondary;
```

Comment:
Could you add `BookmarkSignal` to `SignalStories` so we have a regression test? Technically, this should have broken the test and required approval for Design to continue (who obviously would approve it because they wanted it!)